runatlantis.io/docs/server-configuration.md

```diff
@@ -447,6 +447,16 @@ and set `--autoplan-modules` to `false`.
 
   This is useful when running multiple Atlantis servers against a single repository.
 
+### `--hide-unchanged-plan-comments`
```

Comment:
Would you test this out in your own setup end to end to ensure this is working as expected with and without the flag? It would be nice to see screen shots

Reply:
ok, no problem! I will return with screenshots tomorrow

Reply:
here are some screenshots
1 / 6 project with changes
<img width="855" alt="image" src="https://user-images.githubusercontent.com/10697000/221365857-6926e58b-4f8f-48af-a8fe-e5fcdb0123dc.png">

two projects, no changes, nothing to show
<img width="857" alt="image" src="https://user-images.githubusercontent.com/10697000/221365929-d4ab93f4-7c96-40ca-953c-fcf27f84a1b2.png">


also, fixed command-line flags

Reply:
@nitrocode also, I can add message like this 
```
Plans with `No changes` are hidden.
```

Reply:
Nice!

- How about when the flag is not set? Does it maintain the same results?
- What about the PR status checks at the bottom of the PR? Do those checks still show the appropriate PR summary (`No changes`, `0 to add, 1 to change...`, etc) status on plan?

Reply:
Also please include the screenshots in the PR body. I created this thread so it would be easier to discuss. Once the comment is resolved, finding the screenshots will be difficult.

Reply:
PR status checks still show everything

without flag output is:
<img width="853" alt="image" src="https://user-images.githubusercontent.com/10697000/221367188-f1df171f-8120-412e-abd0-7b20a080cfbf.png">

when planning >50 projects it's easy to miss some changes, plans are monotonous

Reply:
PR body updated

Reply:
Thank you!